docs/reference/modules/i18n.md

```diff
@@ -46,7 +46,7 @@ Other notes:
 - Two or more locales may not be registered with the same `hostname` *and* the same path `prefix`. Apostrophe will throw an error in this case.
 - The **default locale** (either the `defaultLocale` setting or the first registered locale) does not need a `hostname` or path `prefix` setting.
 - If *any* locale has a `hostname` setting *one* of these must be true:
-  1. **The Apostrophe app must have a `baseUrl` set in the `app.js` file** (a best practice in most cases anyway), OR
+  1. **The Apostrophe app must have a `baseUrl` set in the `app.js` file** (a best practice in most cases anyway) or set through the `APOS_BASE_URL` environment variable, OR
```

Comment:
`app.js` is not really best practice because that'll break local development which has a different URL. `baseUrl` is usually set in production, either via the new environment variable (which most people will prefer) or in a `data/local.js` file.